cdk_infra/lib/config/cluster-config/clusters.yml

```diff
@@ -91,11 +91,11 @@ clusters:
     launch_type: ec2
     instance_type: m6g.large
     cert_manager: true
-  - name: dev-collector-ci-amd64-1-22
+  - name: dev-collector-ci-amd64-1-23
     version: "1.22"
     launch_type: ec2
     instance_type: "m5.large"
-  - name: dev-collector-ci-arm64-1-22
+  - name: dev-collector-ci-arm64-1-23
     version: "1.22"
```

Comment:
```suggestion
    version: "1.23"
```

Reply:
ty, trying to do too many things at once right now.